stream/web-sdk/v3.md

```diff
@@ -0,0 +1,744 @@
+# Web SDK
```

Comment:
We are missing the web-sdk changelog https://transaction-stream.fidel.uk/docs/web-sdk-changelog.. unless we move it to a different place, are we?